src/components/member-profile/contribution/index.js

```diff
@@ -135,6 +152,13 @@ const ContributionCard = ({
           )}
         </div>
         <div className={classNames.rightSection}>
+          {isNoteworthy !== undefined ? (
```

Comment:
Can we move this check inside `SuperUserOptions` component? We are already passing `isNoteworthy` prop

Reply:
I want to check if it is undefined and based on that I want to render it, because it should show up only for the tasks and not for the PR's